easybuild/easyconfigs/f/FreeFem++/FreeFem++-3.58-foss-2017b-downloaded-deps.eb

```diff
@@ -13,26 +13,35 @@ toolchainopts = {'usempi': True, 'pic': True}
 
 source_urls = ['http://www.freefem.org/ff++/ftp/']
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['e205c0a89fe8edc876d97b8a6ba43825b83c7f8dbdd49fdf972fa1f089b62743']
+patches = ['%(name)s-%(version)s_ebvars.patch']
+checksums = [
+    'e205c0a89fe8edc876d97b8a6ba43825b83c7f8dbdd49fdf972fa1f089b62743',  # freefem++-3.58.tar.gz
+    'e1fe8b26ab417e059653538bcd76214ca0ff4fc5486af11014413335de1ba58d',  # FreeFem++-3.58_ebvars.patch
+]
 
-builddependencies = [('Autotools', '20170619')]
+builddependencies = [
+    ('Autotools', '20170619'),
+    ('flex', '2.5.39'),
+    ('Bison', '3.0.4'),
+    ('CMake', '3.10.1'),
+]
 
 dependencies = [
     ('HDF5', '1.10.1'),
     ('GSL', '2.4'),
     ('freeglut', '3.0.0'),
 ]
 
-preconfigopts = "autoreconf -i &&"
-
-configopts = '--with-hdf5="$EBROOTHDF5"/bin '
+configopts = '--with-hdf5="$EBROOTHDF5"/bin/h5pcc '
```

Comment:
No need for the double qoutes around `$EBROOTHDF5`: `--with-hdf5=$EBROOTHDF5/bin/h5pcc` .

Also, why is this not handled by the `FreeFem++` easyblock in https://github.com/easybuilders/easybuild-easyblocks/pull/1488?

If there's an easyblock, the easyconfig should be as minimal as possible...

Same for rest of `configopts` & also `prebuildopts` below...

Reply:
@boegel Do you want me to put the `runtest`, `sanity_check_path`, and `modextrapaths` as well in the easyblock?

Reply:
Yes, that makes most sense to me...

Reply:
OK.